src/core/public/chrome/ui/header/header.scss

```diff
@@ -110,3 +110,11 @@
 #globalHeaderBars:has(.primaryApplicationHeader) {
   border-bottom: 1px solid $euiColorLightShade;
 }
+
+/* Control flex items to wrap when the viewport is less than the medium size */
+@media (max-width: 768px) {
```

Comment:
nit: are $euiBreakpoints available here? If so, could use `$euiBreakpoints['m']` instead of the hard coded value